public/index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="utf-8" />
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <meta name="theme-color" content="#000000" />
+    <meta name="theme-color" content="#777777" />
```

Comment:
I see this PR changes the readme and theme color. It is better to do it in different PRs. Please create another PR and move the changes for color to it.